airflow/providers/yandex/operators/yandexcloud_dataproc_operator.py

```diff
@@ -0,0 +1,334 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+from airflow.providers.yandex.hooks.yandexcloud_dataproc_hook import DataprocHook
+from airflow.providers.yandex.operators import YandexCloudBaseOperator
+from airflow.utils.decorators import apply_defaults
+
+
+class DataprocCreateClusterOperator(YandexCloudBaseOperator):
```

Comment:
Is there any reason for this inheritance? `YandexCloudBaseOperator` does not implement any common methods. In my opinion, adding 3 lines in all operators is a small cost for better encapsulation :)

Reply:
Done. Removed the base class